Determinar si dos digitos son iguales

Yerlis09 opened the pull request:
creacion de la funcionalidad que determina si
    dos digitos son iguales. porque se requeria una
    funcionalidad que luego de haber ingresado un numero
    entero de dos digitos retornara, si eran o no iguales

    tarea: #10

Yerlis09 committed:
feat: determinar cual es el numero mayor de dos digitos

creacion de la funcionalidad que permite determinar a
partir de dos enteros, cual es el mayor. porqué el cliente
solicitó que el proyecto tuviera esta funcionalidad

tarea: #11

Yerlis09 committed:
fix: corregir validaciones

corregir en la funcionalidad determinar
cual es el mayor de dos numeros. porque
hubo una equivocación cuando se hace la
tercera validacion, la cual deberia
validar si el tipo de dato de un numero
no sea un string

resuelve: #11

Yerlis09 committed:
feat: determinar si dos digitos son iguales

creacion de la funcionalidad que determina si
dos digitos son iguales. porque se requeria una
funcionalidad que luego de haber ingresado un numero
entero de dos digitos retornara, si eran o no iguales

tarea: #10

oscles commented:
El commit tiene una redaccion clara, pero no estas usando las convenciones.

oscles commented on 1_algoritmo.py:
```diff
@@ -118,4 +118,43 @@ def obtener_catidad_digitos(numero = 0):
+
+# Leer un numero entero de dos digitos y determinar si los dos digitos son iguales
+
+def determinar_si_dos_digitos_son_iguales(numero = 0):
```
En esta tarea podias tomar un enfoque de solo tomar numero de 2 digitos y con base a eso hacer el algoritmo.

oscles commented on 1_algoritmo.py:
```diff
@@ -118,4 +118,43 @@ def obtener_catidad_digitos(numero = 0):
+        return 'Ingrese un numero entero mayor a 9'
+    else:
+        str_numero = (str(numero))
+        if (str_numero[0] == str_numero[1]):
```
Aqui lo mismo haceruna condicion es ams sencillo y tiene mejor rendimiento

Yerlis09 committed:
refactor: Corregir validaciones

Corregir el algoritmo. el cliente solicito
que hiciera unas modificaciones en el algoritmo,
ya que no cumplia con lo requerido

resuelve : #11

Yerlis09 committed:
chore: Determinar si dos digitos son iguales

agregar las convenciones del commit, porque
no se estaban utilizando y despues puede ser
un problema, para los compañeros que esten
trabajando en el mismo proyecto

Resuelve: #12
